cng/aes.go

```diff
@@ -151,38 +119,45 @@ func (c *aesCipher) NewGCMTLS() (cipher.AEAD, error) {
 	return newGCM(c.key, true)
 }
 
-type aesCBC struct {
-	kh      bcrypt.KEY_HANDLE
-	iv      [aesBlockSize]byte
-	encrypt bool
+type cbcCipher struct {
```

Comment:
This could be moved to `cipher.go`, but I kept it here to minimize PR diffs. Will improve this, together with other cipher refactors, in a follow up PR.